test/gtest/uct/uct_test.h

```diff
@@ -189,15 +189,15 @@ class uct_test : public testing::TestWithParam<const resource*>,
     virtual void init();
     virtual void cleanup();
     virtual void modify_config(const std::string& name, const std::string& value);
-    virtual void get_config(const std::string& name, std::string& value);
+    virtual bool get_config(const std::string& name, std::string& value);
```

Comment:
i think would be better to throw exception (e.g UCS_TEST_FAIL) rather than returning a value
(as general notice, the C++ way of handling error is to throw , and not return error code)